src/mscorlib/src/System/Diagnostics/Stackframe.cs

```diff
@@ -2,25 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
 using System.Text;
-using System;
-using System.IO;
 using System.Reflection;
 
 namespace System.Diagnostics
 {
-    // There is no good reason for the methods of this class to be virtual.
```

Comment:
I think we should split the cleanup off to its own PR to make sure it doesn't get squashed together with the other changes during checkin. It helps to avoid having a few important changes hidden amongst numerous refactorings.

Reply:
Simplified just to this change